composer/core/state.py

```diff
@@ -136,6 +136,8 @@ def get_fsdp_sharded_optim_state_dict(full_optim_state_dict: Dict[str, Any], mod
     if version.parse(torch.__version__) < version.parse('1.13.0'):
         raise RuntimeError('To use FSDP with Composer, you must use torch>=1.13.0.')
     from torch.distributed.fsdp import FullyShardedDataParallel as FSDP
+    log.debug(
```

Comment:
do you mean to keep all these log.debug statements?

Reply:
Let's keep them imo

Reply:
yeah, i try to differentiate between prints (added as one offs and dont intend to keep) and `log.debug` (added both because they are useful for me, and potentially useful for someone else in the future)